### PR TITLE
Style fixes

### PR DESCRIPTION
**Pull request by tkruse**

### CODE REVIEW

File: CppCoreGuidelines.md [hunk 1]
```diff
@@ -10385,7 +10385,7 @@ Readability and safety.
 
 As an optimization, you may want to reuse a buffer as a scratch pad, but even then prefer to limit the variable's scope as much as possible and be careful not to cause bugs from data left in a recycled buffer as this is a common source of security bugs.
 
-    {
+    void writeFile() {
```
Comment (blakehawkins) [inline CppCoreGuidelines.md]: Based on a quick grep, looks like `snake_case` would be better here for consistency.

Comment (tkruse) [inline CppCoreGuidelines.md]: I guess you are right, I forgot to check that. For consistency, should the inner calls also be snake-case, then? In the following lines, I mean.

Comment (tkruse) [inline CppCoreGuidelines.md]: fixed, also in method body, also renamed simpleFunc to simple_funx in another place. I also saw functions myX and onClick, left those alone.

**Comment by gdr-at-ms** [comment]
squash commits?

**Comment by tkruse** [comment]
rebased, squashed

**Pull request merged**